test/api-test.js

```diff
@@ -50,6 +50,33 @@ describe('RippleAPI', function() {
       _.partial(checkResult, responses.preparePayment, done));
   });
 
+  it('preparePayment - 2', function(done) {
+    this.api.preparePayment(address, requests.preparePayment2, instructions,
+      _.partial(checkResult, responses.preparePayment2, done));
+  });
+
+  it('preparePayment - 3', function(done) {
+    const paymentRequest = _.cloneDeep(requests.preparePayment2);
+    paymentRequest.source.amount.currency = 'USD';
+    paymentRequest.destination.amount.currency = 'LTC';
+    const expected = _.cloneDeep(responses.preparePayment2);
```

Comment:
I'm thinking it might be better to just make new fixtures for this case even if there is some redundancy in the fixtures since it will get rid of all these lines of code.